crates/prelude/src/lib.rs

```diff
@@ -61,14 +61,14 @@ cfg_if! {
         /// Collection types.
         pub mod collections {
             pub use self::{
+                BinaryHeap,
                 BTreeMap,
                 BTreeSet,
-                BinaryHeap,
                 LinkedList,
                 VecDeque,
             };
-            pub use alloc::collections::*;
             pub use core::ops::Bound;
+            pub use alloc::collections::*;
```

Comment:
actually this is in wrong order. I wonder if our formatting CI actually checks these lines of code since they are conditionally compiled code.

Reply:
why? 
My understanding doesn't need any order
for example 
use A
use B
It's the same
use B
use A

I just need to be consistent

Reply:
our code guidelines demand alphabetically sorted import statements. the only problem right now is that for whatever reasons `rustfmt` does not automatically check this region of code. We need to inspect why. Probably it is because this part of the codebase is only conditionally compiled.

Reply:
ok, delete it